build.rs

```diff
@@ -240,13 +240,13 @@ mod generate {
             as_enum.write_all(b"x => Self::Unrecognized(x)\n")?;
 
             as_enum.write_all(b"}}\n")?;
-            as_enum.write_all(b"pub(crate) fn into_u64(&self) -> u64 {\n")?;
+            as_enum.write_all(b"pub(crate) fn into_u64(self) -> u64 {\n")?;
```

Comment:
It seems that build.rs was out of sync with the checked in source.

Reply:
Could use some CI to ensure they're in sync :)

Reply:
Yeah. I don't think I will do that myself though.